ce/tests/test_api.py

```diff
@@ -168,28 +183,67 @@ def test_data_bad_time(populateddb):
     assert 'time parameter "time-not-an-int" not convertable to an integer.' == \
         str(exc.value)
 
-@pytest.mark.parametrize(('variable'), ('tasmax', 'tasmin'))
-def test_data_single_variable_file(populateddb, variable):
-    rv = data(populateddb.session, 'cgcm3', 'rcp45', 1, None, variable,
-              timescale='monthly')
+
+@pytest.mark.parametrize('variable', (
+        'tasmax',
+        'tasmin',
+))
+@pytest.mark.parametrize('timescale, time_idx, expected_ymd', (
+        ('monthly', 8, (1985, 9, 15)),
+        ('seasonal', 2, (1985, 7, 15)),
+        ('yearly', 0, (1985, 7, 2)),
+))
+def test_data_single_file(populateddb, variable,
```

Comment:
Do we do any exception testing here? E.g. what happens if you give `time_idx=1` for `timescale='yearly'`, etc.?